todo_app/tests/test_unit.py

```diff
@@ -0,0 +1,16 @@
+from todo_app.todo import Item
+from todo_app.view_model import ViewModel
+
+def test_todo_items():
+    items_to_do = [Item(2344,"Answer emails", "To do"), Item(2345,"Do exercise", "In Progress") , Item(2346,"Go Home", "Done")]
+    VM = ViewModel(items_to_do)
```

Comment:
Following the Python naming convention, this variable could be called `vm` or `view_model` and `ToDoItems` should be renamed to `to_do_items`.